easycv/datasets/classification/pipelines/auto_augment.py

```diff
@@ -157,6 +158,14 @@
         direction='vertical')
 ]
 
+three_augment_policies = [[
```

Comment:
modify to augment_policies, remove three

Reply:
done
merge ThreeAugment into MMAutoAugment